runtime/runtime_impl_jetty12/src/main/java/com/google/apphosting/runtime/jetty/JettyServletEngineAdapter.java

```diff
@@ -118,27 +120,45 @@ public InvocationType getInvocationType() {
             return InvocationType.BLOCKING;
           }
         };
-    rpcConnector =
-        new DelegateConnector(server, "RPC") {
-          @Override
-          public void run(Runnable runnable) {
-            // Override this so that it does the initial run in the same thread.
-            // Currently, we block until completion in serviceRequest() so no point starting new
-            // thread.
-            runnable.run();
-          }
-        };
-    server.addConnector(rpcConnector);
+
+    // Don't add the RPC Connector if in HttpConnector mode.
```

Comment:
Hum. so we did that, what the side effect? 2 connectors and one not used? More memory used?

Reply:
Probably, but I don't think the memory saving here is that significant considering the `DelegateConnector` was never used.